migrations/0021-add-userFeedback-table.ts

```diff
@@ -0,0 +1,64 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class AddUserFeedbackTable1602394486221 implements MigrationInterface {
```

Comment:
Kinda annoying but we gotta make sure the migration timestamps are in order; the enum-to-strings migration has timestamp 1602914093929.